From 67859175c4c5e025499f54c73d4d769a0b7785cd Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Tue, 8 Feb 2022 23:20:43 +0100
Subject: [PATCH] security_policy.conf: Add /usr/palm/applications

So that apps actually render something and don't show network connection error.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/launch/security_policy.conf | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/launch/security_policy.conf b/files/launch/security_policy.conf
index cc29f96..6251de3 100644
--- a/files/launch/security_policy.conf
+++ b/files/launch/security_policy.conf
@@ -8,7 +8,8 @@
 7/path=/var/palm/license/
 8/path=/usr/palm/license/
 9/path=/usr/palm/plugins/license/
-size=9
+10/path=/usr/palm/applications/
+size=10
 
 [Trusted]
 1/path=/media/
